spring-cloud-stream/src/main/java/org/springframework/cloud/stream/annotation/StreamListener.java

```diff
@@ -43,10 +52,17 @@
  * <li>The value set on the {@link org.springframework.messaging.handler.annotation.SendTo} annotation, if present</li>
  * </ul>
  *
+ * <h3>Declarative mode</h3>
+ *
+ * The annotation may also have an empty value (the default), case in which the method is a
```

Comment:
something not quite right with ", case in which" (maybe you meant "for the case" or something similar?)